### DISCUSSION TITLE
Error MinecraftServer.class when run - java: cannot access net.minestom.server.MinecraftServer

### DISCUSSION THREAD

**Discussion by LabrixSpigotMC** [Minestom/Minestom#1492] [Help] [answered]
`java: cannot access net.minestom.server.MinecraftServer
  bad class file: .m2/repository/com/github/Minestom/Minestom/057109c94e/Minestom-057109c94e.jar!/net/minestom/server/MinecraftServer.class
    class file has wrong version 61.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.`

**What can I do to correct the error?**

**Comment by HSGamer** [comment, marked as answer]
try running it in Java 17

**Reply by LabrixSpigotMC** [reply]
thank you very much :)
